src/compose/ComposeBox.js

```diff
@@ -259,11 +259,11 @@ class ComposeBoxInner extends PureComponent<Props, State> {
         try {
           response = await api.uploadFile(auth, attachments[i].uri, fileName);
         } catch (e) {
-          showToast(_('Uploading {fileName} failed.', { fileName }));
+          showToast(_('Failed to upload file: {fileName}.', { fileName }));
           this.setMessageInputValue(
             this.state.message.replace(
               placeholder,
-              `[${_('Uploading {fileName} failed.', { fileName })}]()`,
+              `[${_('Failed to upload file: {fileName}.', { fileName })}]()`,
```

Comment:
Let's also drop the `.` at the end. That works when the filename is in the middle of the sentence, but here it risks looking like part of the filename; and in general I think we don't put periods at the end of these short messages.